web/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Is het tijd voor retrospective?</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/js/bootstrap.min.js" integrity="sha384-ODmDIVzN+pFdexxHEHFBQH3/9/vQ9uori45z4JjnFsRydbmQbmL5t1tQ0culUzyK" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.5/dist/umd/popper.min.js" integrity="sha384-Xe+8cL9oJa6tN/veChSP7q+mnSPaj5Bcu9mPX5F5xIGE0DVittaqT5lorf0EI7Vk" crossorigin="anonymous"></script>
+</head>
+<body>
+<center>
+    <h1 class="mt-5">
+        Is het tijd voor een retrospective?
+    </h1>
+    <div class="form-text">
+        Kies een van de onderstaande opties voor een retrospective template.
+    </div>
+    <div class="mt-5">
+        <a class="btn btn-primary " href="/random">
+            Ik gok er op!
+        </a>
+        <a class="btn btn-secondary disabled" href="/list">
+            Ik kies zelf wel.
+        </a>
+        <div class="form-text">
+            Krijg je geen nieuwe template? Probeer een hard refresh! ( <a id="platform"> Control/Command </a> + Shift + R)
+        </div>
+    </div>
+
+</center>
+
+
+
+<footer class="b position-absolute" style="bottom: 0;">
+Want to contribute to this project? Click <a href="https://github.com/marnixah/ishettijdvoorretrospective">here!</a>
+</footer>
+
+<script>
+
+    function getOS() {
+        let os
+
+        if (navigator.appVersion.indexOf("Win") != -1) os =
+            "Windows OS";
+        if (navigator.appVersion.indexOf("Mac") != -1) os =
+            "MacOS";
+        if (navigator.appVersion.indexOf("X11") != -1) os =
+            "UNIX OS";
+        if (navigator.appVersion.indexOf("Linux") != -1) os =
+            "Linux OS";
+
+        return os;
+    }
+
+    let os = getOS()
+    let oselement = document.getElementById("platform")
+
+    if (os === 'Windows') {
```

Comment:
This check will always fail, no?

Reply:
No, this actually works. Atleast for MacOS, have yet to try it for Windows/Linux. Although if it does not work, it won't break the website or server.

Reply:
```js
os = "Windows OS";

if (os === 'Windows') {
```
How would this work?

Reply:
Shouldn't use windows then.